src/mutations/user/putWallet.ts

```diff
@@ -96,7 +96,7 @@ const resolver: MutationToPutWalletResolver = async (
     wallet = await atomService.update({
       table,
       where: { id, userId: viewer.id },
-      data: { address },
+      data: { address, updatedAt: new Date() },
```

Comment:
`where : { ..., archived: false }` ?

Reply:
already did the [checking before](https://github.com/thematters/matters-server/blob/151ce56079d809497ead73fe2fc6f7ed04b62af1/src/mutations/user/putWallet.ts#L87-L94), should be good?

Reply:
but this is going to update both `archived = true or archived = false`